app/src/main/res/layout/screen_monthly_screening_report_list.xml

```diff
@@ -81,6 +80,16 @@
       android:layout_height="wrap_content"
       android:text="@string/monthly_screening_reports_screening_report" />
 
+    <TextView
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:text="@string/monthly_screening_reports_submitted_with_date_and_time" />
+
+    <TextView
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:text="@string/monthly_screening_reports_submitted_with_date" />
+
```

Comment:
[non-blocking] Which UI elements do these correspond to?

Reply:
@prabhanshuguptagit these are dummy text, so as the lint error is succeeded and we can then send these strings to transifex before we do the actual implementation.

Reply:
👍🏼